eos_data_distribution/ndn/dbus/base.py

```diff
@@ -251,7 +251,9 @@ def __init__(self, name, dbus_name, skeleton):
         self._dbus_name = build_dbus_name(dbus_name, name)
         self._interface_skeleton_object = skeleton
 
-        self.con = Gio.bus_get_sync(BUS_TYPE, None)
+        address = Gio.dbus_address_get_for_bus_sync(BUS_TYPE, None)
+        self.con = Gio.DBusConnection.new_for_address_sync(address, Gio.DBusConnectionFlags.NONE, None, None)
```

Comment:
This is really odd! Do things not work without it? I'd expect the address to return exactly the same value every time.